cmd/ddev/cmd/config.go

```diff
@@ -258,24 +265,20 @@ func init() {
 
 // getConfigApp() does the basic setup of the app (with provider) and returns it.
 func getConfigApp(providerName string) (*ddevapp.DdevApp, error) {
-	// Find app root
-	cd, err := os.Getwd()
+	appRoot, err := os.Getwd()
 	if err != nil {
-		return nil, err
+		return nil, fmt.Errorf("could not determine current working directory: %v", err)
 	}
 
-	// Check for an existing config in this or a parent dir
-	appRoot, err := ddevapp.CheckForConf(cd)
-	if err != nil {
-		// An error indicates no config file was found, use current directory
-		appRoot = cd
+	// Check for an existing config in a parent dir
+	otherRoot, _ := ddevapp.CheckForConf(appRoot)
+	if otherRoot != appRoot {
+		util.Error("Is it possible you wanted to `ddev config` in parent directory %s?", otherRoot)
```

Comment:
This confirmation isn't working for me. When I run `ddev config` in a new directory, I get this warning:

```
Is it possible you wanted to `ddev config` in parent directory ?
```

Note the empty value being passed to the format string. I can walk up the directory stack and get the warning in every directory (`~/junk/abc/def`, `~/junk/abc`, `~/junk`), except the home directory, where it fails with:

```
Is it possible you wanted to `ddev config` in parent directory ?
could not create new config: ddev config is not useful in home directory (/Users/andrew)
```